File: 1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
  for (let i = 0; i < nums.length; i++) {
    for (let j = i+1; j < nums.length; j++) {
      let isTotal = nums[i] + nums[j] === target;
      if (isTotal) {
        return [i, j];
      }
    }
  }
  return [];
};

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        complement = {}
        for i in range(len(nums)): 
            if(complement.get(nums[i]) != None):
                return [complement.get(nums[i]), i]
            else:
             comp = target - nums[i]
             complement[comp] = i
        return []